test-end-to-end/webapp-cypress/cypress/e2e/004_001_assignment_creation_submission_grading.cy.js

```diff
@@ -0,0 +1,104 @@
+import setDate from '../support/setDate'; //https://github.com/cypress-io/cypress/issues/1366#issuecomment-437878862
```

Comment:
I don't think this is being used, we're using our own setDate. If not used, please remove. If there is a related dependency, please remove that also.